src/endpoints.py

```diff
@@ -0,0 +1,36 @@
+from flask_restful import Resource
+from .search import client
+from .ingest import fetch_information_models, fetch_concepts, fetch_dataservices, fetch_datasets
+from .search.responses import SearchResponse
+
+
+class Search(Resource):
+    def post(self):
+        result = client.search_all("")
+        return SearchResponse().map_response(result)
+
+
+class Count(Resource):
+    def get(self):
+        return client.count()
+
+
+class Update(Resource):
+    def post(self):
```

Comment:
Likar ikkje at funksjonen for update (put i http) heiter post (som er http-verbet for create).